lxc_ssh.py

```diff
@@ -523,6 +523,19 @@ def __init__(self, play_context, new_stdin, *args, **kwargs):
         self.control_path = None
         self.control_path_dir = None
 
+    def _set_command_prefix(self):
+        # check for cgroupv2 and use systemd-run to run the commands if needed
+        (returncode_cgroup, stdout_cgroup, stderr_cgroup) = self._exec_command(
+            "stat /sys/fs/cgroup/cgroup.controllers", None, False
```

Comment:
Actually, I meant [Python stat](https://docs.python.org/3/library/stat.html).
Or is there a need to execute an external command?

Reply:
Can't use `os.stat()` unless I completely misunderstood where the code runs.
As I can see (and tests seemed to imply this), this code runs on the computer running `ansible-playbook`. Which means that the file won't necessarily exist (and if it did, it wouldn't tell us much).
For example, using this in the body of `_set_command_prefix()`:
```python
        cgroup2 = os.path.exists("/sys/fs/cgroup/cgroup.controllers")
```

This will evaluate locally and, since I'm running on macOS, wouldn't find the file. But the target computer (accessed with ssh) has Linux with cgroupv2 and is running the containers.

Reply:
And of course you are right. That's my fault, didn't think this through.